lib/sidekiq/web/helpers.rb

```diff
@@ -173,7 +173,7 @@ def display_args(args, truncate_after_chars = 2000)
     end
 
     def csrf_tag
-      "<input type='hidden' name='authenticity_token' value='#{session[:csrf]}'/>"
+      "<input type='hidden' name='authenticity_token' value='#{session[:csrf]}'/>" if session
```

Comment:
I don't believe this should ever be disabled.  Can you explain why this is necessary?